examples/dns-and-glb/versions.tf

```diff
@@ -1,7 +1,29 @@
+#####################################################
```

Comment:
Activity Tracker reference does not seem to be accurate
Copyright is not needed